docs/agents/letterAgent.md

```diff
@@ -0,0 +1,35 @@
+LetterAgent is an agent added as example to show how to add new agents to system. 
```

Comment:
Write something about how we can use this agent (cities game and something else)